### PR TITLE
[RISCV] Convert all floating point vector type operands to integer vector type

### PR DESCRIPTION
**Pull request by 4vtomat**

### CODE REVIEW

File: llvm/lib/Target/RISCV/RISCVInstrInfoXSf.td [hunk 1]
```diff
@@ -450,6 +474,18 @@ multiclass VPatVC_XVV<string intrinsic_suffix, string instruction_suffix,
                      "PseudoVC_V_" # instruction_suffix # "_" # vti.LMul.MX,
                      wti.Vector, wti.Vector, vti.Vector, type, vti.Log2SEW,
                      wti.RegClass, vti.RegClass, kind, op1_kind>;
+  // Add another patterns for float type return value.
+  if !ne(wti.SEW, 8) then {
+    defvar wfti = !cast<VTypeInfo>("VF"#wti.SEW#wti.LMul.MX);
```
Comment (topperc) [inline llvm/lib/Target/RISCV/RISCVInstrInfoXSf.td:479]: Do we need new tests for this?

Comment (4vtomat) [reply llvm/lib/Target/RISCV/RISCVInstrInfoXSf.td:479]: Yes, I'll add some tests for it~

Comment (4vtomat) [reply llvm/lib/Target/RISCV/RISCVInstrInfoXSf.td:479]: Do you think test cases all of combination of vector type arguments(including float and int) are needed?
Or just provide test cases for some mix type arguments are enough.

Comment (topperc) [reply llvm/lib/Target/RISCV/RISCVInstrInfoXSf.td:479]: some mix is probably good enough

**Comment by 4vtomat** [comment]
ping

**Review by topperc — APPROVED** [review]
LGTM

**Pull request merged**